MathlibTest/Algebra/MonoidAlgebra/Defs.lean

```diff
@@ -0,0 +1,46 @@
+import Mathlib.Algebra.MonoidAlgebra.Defs
+
+variable {R M A} [Semiring R] [Monoid M] [AddMonoid A]
+
+
+section Notation
+open scoped MonoidAlgebra AddMonoidAlgebra
+
+-- TODO: could resolve ambiguity based on Monoid / AddMonoid
+/--
+error: Ambiguous term
+  R[M]
+Possible interpretations:
+  R[M] : Type (max ?u.29 ?u.32)
+  ⏎
+  MonoidAlgebra R M : Type (max ?u.29 ?u.32)
```

Comment:
@YaelDillies, was it a deliberate choice not to add a `app_unexpander` for `MonoidAlgebra`?